.storybook/preview-head-vrt.html

```diff
@@ -3,10 +3,16 @@
 	#root *,
```

Comment:
Please review the notes in **Relevant technical choices** in the PR description.